bin/anvi-run-ncbi-cogs

```diff
@@ -47,12 +49,14 @@ if __name__ == '__main__':
 
     parser = ArgumentParser(description=__description__)
 
-    parser.add_argument(*anvio.A('contigs-db'), **anvio.K('contigs-db'))
+    parser.add_argument('-c', '--contigs-db', metavar='CONTIGS_DB', required=False,
+                        help="Anvi'o contigs database generated by 'anvi-gen-contigs-database'")
     parser.add_argument(*anvio.A('cog-version'), **anvio.K('cog-version'))
     parser.add_argument(*anvio.A('cog-data-dir'), **anvio.K('cog-data-dir'))
     parser.add_argument(*anvio.A('num-threads'), **anvio.K('num-threads'))
     parser.add_argument(*anvio.A('temporary-dir-path'), **anvio.K('temporary-dir-path'))
 
+    parser.add_argument('--aa-file', default=None, metavar="AA_FILE", help="An amino acid fasta file to annotate.")
```

Comment:
With this PR you in fact implement two different entry routes to the program that are not compatible. One is the old way of providing a contigs-db, and the other is used when the user provides a FASTA file. I think it would be most helpful to change the help menu in such a way it is clear. For some inspiration, please tak ea look at the output of,

```
anvi-estimate-metabolism -h
```

and input options there.

I think the input option for FASTA should also require an output file name, so the output is not always named `cog_functions.tsv`, and give user control.

Reply:
Cleaned up the help menu as suggested (much better). An added what should've been an obvious output file path.